<ui version="4.0" >
 <class>DasdcatClass</class>
 <widget class="QWidget" name="DasdcatClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dasdcat</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>521</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title" >
    <string>Input</string>
   </property>
   <widget class="QLineEdit" name="filename" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>30</y>
      <width>381</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="browseFileButton" >
    <property name="geometry" >
     <rect>
      <x>490</x>
      <y>30</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Ckd File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="shadowFile" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>70</y>
      <width>381</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="browseSfButton" >
    <property name="geometry" >
     <rect>
      <x>490</x>
      <y>70</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Shadow file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Dataset Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dsname" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>110</y>
      <width>381</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>200</y>
     <width>521</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title" >
    <string>Run Options</string>
   </property>
   <widget class="QLineEdit" name="memberName" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>120</y>
      <width>351</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Membe name</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="listRadioButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>List PDS members</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="specificRadioButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Show specific member content</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="allRadioButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Show contest of all members in PDS</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitButton" >
   <property name="geometry" >
    <rect>
     <x>450</x>
     <y>390</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>390</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Run</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
